mozilla/activity-stream
# Bug 1520925 - Further refine CardGrid (WIP)

# QA Steps

Set `browser.newtabpage.activity-stream.discoverystream.config` to `{"enabled":true,"layout_endpoint":"https://gist.githubusercontent.com/gvn/76af39c92085c247dd5dd7ccc2412618/raw/a3b7017d292f1a3e0b709f8b660d2dbd577a85ba/disco.json"}`

Open new tab and see that:

- CardGrid with 3 items is 3 columns
- CardGrid with 9 items is 3 columns
- CardGrid with 12 items is 4 columns
- CardGrid with 4 items is 4 columns
- CardGrids have titles

Based on [Layouts Figma](https://www.figma.com/file/B1eynmsVDYKdpDWZhP8xYBWq/User-Research-Layout-Prototypes?node-id=20%3A135)
